spec/unit/provider/oneview_uplink_set_c7000_spec.rb

```diff
@@ -18,18 +18,17 @@
 require_relative '../../support/fake_response'
 require_relative '../../shared_context'
 
-provider_class = Puppet::Type.type(:oneview_uplink_set).provider(:oneview_uplink_set)
+provider_class = Puppet::Type.type(:oneview_uplink_set).provider(:c7000)
 resourcetype = OneviewSDK::UplinkSet
```

Comment:
Should add something along the lines of the following code to account for different APIs
```ruby
api_version = login[:api_version] || 200
resource_name = 'UplinkSet'
resourcetype = if api_version == 200
                 Object.const_get("OneviewSDK::API#{api_version}::#{resource_name}")
               else
                 Object.const_get("OneviewSDK::API#{api_version}::C7000::#{resource_name}")
               end
```